jeffreywallphd/OpenFinAL
# Fall2024 dev combined ba5200 mis4000

### Description
Feature: Stock comparison based on various metrics.
Input: User provides stock symbols.
Output: Stock comparison based on chosen metrics (e.g., P/E ratio, market cap, etc.).
Export Options: Users can export the comparison results as CSV or PDF.

###Changes
- Added a stock search input where users can enter multiple stock symbols.
- Implemented functionality to fetch stock data using an API.
- Created a comparison dashboard showing key stock metrics.
- Implemented the ability to export the comparison data as CSV and PDF files.
- Added validation for correct stock symbols and error handling for invalid inputs.

### How to Test
1 Start the application and navigate to the stock comparison section.
2. Input one or more stock symbols (e.g., "AAPL", "GOOG", "AMZN").
3. Click "Search" to view the stock comparison metrics (P/E, market cap, etc.).
4. Ensure the comparison data is displayed correctly.
5. Test the export options:
   - Export the data as CSV and ensure the correct format.
   - Export the data as PDF and ensure the format is readable and includes all metrics.
6. Verify that the stock comparison updates correctly when different symbols are entered.
